Mathlib/Data/Fintype/Basic.lean

```diff
@@ -1217,7 +1217,7 @@ end Trunc
 
 namespace Multiset
 
-variable [Fintype α] [DecidableEq α] [Fintype β]
+variable [Fintype α] [DecidableEq α] [Fintype β] [DecidableEq β]
```

Comment:
Either this changed `count_univ` and `map_univ_val_equiv`, or else Lean is automatically purging this hypothesis from those declarations. Either way, this (these?) `DecidableEq` hypotheses shouldn't be included on this line.

Reply:
There was thankfully no incorrectly-typed lemmas here, but this was indeed misleading; I've tidied it up.